lib/coursemology/evaluator/services/evaluate_programming_package_service.rb

```
@@ -109,15 +109,26 @@ def execute_package(container)
   end
 
   def extract_test_report(container)
+    stream = extract_test_report_archive(container)
+
+    tar_file = Gem::Package::TarReader.new(stream)
+    tar_file.each do |file|
+      return file.read
+    end
+  rescue Docker::Error::NotFoundError
```

Do not suppress exceptions.